### PR TITLE
feat: [AXIMST-111] create API for course textbooks

### PR DESCRIPTION
**Pull request by ruzniaievdm**
[AXIMST-111](https://youtrack.raccoongang.com/issue/AXIMST-111)

### CODE REVIEW

File: cms/djangoapps/contentstore/rest_api/v1/views/textbooks.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                    {
+                        "title": "Chapter 1",
+                        "url": "/static/Present_Perfect.pdf"
+                    },
+                    {
+                        "title": "Chapter 2",
+                        "url": "/static/Lear.pdf"
+                    }
+                ],
+                "id": "Textbook_Name"
```
Comment (monteri) [inline cms/djangoapps/contentstore/rest_api/v1/views/textbooks.py:74]: nit: should be tabbed to the right

File: cms/djangoapps/contentstore/rest_api/v1/serializers/textbooks.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    chapters = CourseTextbookChapterSerializer(many=True)
+    tab_title = serializers.CharField()
+
+
+class CourseTextbooksSerializer(serializers.Serializer):
+    """
+    Serializer for representing course's textbooks.
+    """
+
+    textbooks = serializers.ListField()
```
Comment (monteri) [inline cms/djangoapps/contentstore/rest_api/v1/serializers/textbooks.py:32]: I guess it should be
```suggestion
    textbooks = CourseTextbookItemSerializer(many=True)
```
right?

File: cms/djangoapps/contentstore/rest_api/v1/views/textbooks.py (hunk 1 above)
Comment (ruzniaievdm) [reply cms/djangoapps/contentstore/rest_api/v1/views/textbooks.py]: fixed

File: cms/djangoapps/contentstore/rest_api/v1/serializers/textbooks.py (hunk 2 above)
Comment (monteri) [reply cms/djangoapps/contentstore/rest_api/v1/serializers/textbooks.py:32]: Maybe this was intentional, idk

**Pull request merged**